library/ccm.c

```diff
@@ -400,7 +400,6 @@ int mbedtls_ccm_update(mbedtls_ccm_context *ctx,
             mbedtls_xor(ctx->y + offset, ctx->y + offset, local_output, use_len);
 
             memcpy(output, local_output, use_len);
-            mbedtls_platform_zeroize(local_output, 16);
```

Comment:
AFAIUI, this zeroizes the content of `local_output` as soon as it is no longer used and it might not be redundant with the one at the end of the function. The buffer contains the decrypted data and it would keep updated in the loop, so maybe it's OK to remove this line? What do you think? @tom-cosgrove-arm

Reply:
It's fine to remove this line - no need to continually zeroise during the function as we either re-use or do it at the end

Reply:
I think it is redundant in the sense that the buffer will definitely get zeroized before the function exits. Obviously the buffer will now be unzeroized while we are still in the function, but I think that is fine.